spec/services/mentors/create_spec.rb

```diff
@@ -66,6 +66,7 @@
   end
 
   it "schedules participant_added email" do
+    pending "re-enable when m2-release gets the go ahead"
```

Comment:
I'm not sure I follow. Do we not have the go-ahead? If we merge this in, we'll have introduced behaviours into the app the don't want

Reply:
@asmega the base branch for this PR is `m2-release`. I'm going to merge this then seed the app. We do all the testing we want, than before merging this into `develop` I'll remove those.